src/routes/docs/products/messaging/sendgrid/+page.markdoc

```diff
@@ -301,12 +301,10 @@ client
     .setKey('<YOUR_API_KEY>')        // Your secret API key
 ;
 
-const provider = await messaging.updateMailgunProvider(
+const provider = await messaging.updateSendgridProvider(
```

Comment:
Our API has it as `Sendgrid` instead of `SendGrid` 😭

Reply:
Yeahhh
Although, with conventional camelCase and PascalCase, that is considered alright